### DISCUSSION TITLE
Is it possible to obtain sensitivity map without defining FFD?

### DISCUSSION THREAD

**Discussion by Mohsin-Bashir** [mdolab/dafoam#616] [General]
or is it compulsory to have FFD box?

**Comment by friedenhe** [comment]
Nope... You have to define a FFD.

**Comment by Mohsin-Bashir** [comment]
Could you please explain how FFD is used for the calculation of surface sensitivity (sensitivity map)?
From your paper 
![Capturstae](https://github.com/mdolab/dafoam/assets/118594705/69f2ec2c-69dd-44f7-a330-a801aaaf4c14)
_Originally posted by @Mohsin-Bashir in https://github.com/mdolab/dafoam/discussions/544#discussioncomment-7963627_
 what I understood is, FFD is used by pyGeo to deform FFD and give deformed surface nodes. But since I have changed this functionality in my version of code, as I explained here https://github.com/mdolab/dafoam/discussions/576#discussioncomment-8740573, there should not be a requirement to define FFD anymore. **Is there any other place where FFD is used? If yes where and for what?**

**Comment by friedenhe** [comment]
You are right; we really don't need to have an FFD box for sensitivity maps. However, having an FFD is required by DAFoam. It will tell DAFoam that this is a shape variable. We do not actually use the FFD box in sensitivity map calculation.
